.github/workflows/web-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test web code
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/web-test.yml"
+      - "web-admin/**"
+      - "web-cloud/**"
+      - "web-local/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: NPM Install
+        run: npm install
+
+      - name: Disabling telmetry
+        run: cat <<< "$(jq -c -M '.sendTelemetryData |= false' ~/.rill/local.json)" > ~/.rill/local.json
+
+      - name: Prettier checks
+        run: npx prettier --check "**/*"
```

Comment:
I've run into prettier checking `yaml` files in the entire repo formerly. If someone edits examples, they won't get an error, and can merge into main. Then next time someone edits `web` paths, they'll get the broken CI instead. Options:

1. Reconfigure prettier to only check files in web paths
2. Setup prettier as a separate job that runs on all paths